src/webhook/job/manager.go

```diff
@@ -0,0 +1,23 @@
+package job
+
+import (
+	"github.com/goharbor/harbor/src/common/models"
+)
+
+// Manager manages webhook jobs
```

Comment:
please write more details, for example, it manage the job records in DB only right? It does not manage the job by calling the js API correct??

Reply:
> please write more details, for example, it only manages the records in DB only right? It does not manage the job by calling the js API correct??

comments will be add.  it only manages the records in DB